CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## New Features
 
+- PR #787: homogeneity, completeness and v-measure metrics ml-prim
+- PR #724: Entropy metric ml-prim
+- PR #711: Mutual Information metric ml-prim
```

Comment:
I think you have mixed mutual info PR branch also in here?

Reply:
since I needed those prims and they weren't in the main repo yet, i did a git merge on both the mutual info and the entropy branches locally